apis/dataprotection/v1alpha1/backuppolicy_types.go

```diff
@@ -111,6 +111,11 @@ type PodSelector struct {
 	// labelsSelector is the label selector to filter the target pods.
 	*metav1.LabelSelector `json:",inline"`
 
+	// alternateLabelSelector is used to filter available pods when the labelSelector fails.
+	//
+	// +optional
+	AlternateLabelSelector *metav1.LabelSelector `json:"alternateLabelSelector,omitempty"`
```

Comment:
what's the user story (the scenario&problem you want to solve) behind `alternateLabelSelector `

Reply:
You can add some descriptions to the issue or PR description.